apps/api/src/app/events/usecases/send-message/send-message-type.usecase.ts

```diff
@@ -13,12 +14,22 @@ export abstract class SendMessageType {
     message,
     status: 'error' | 'sent' | 'warning',
     errorId: string,
-    errorMessage: string,
+    errorMessageFallback: string | any,
     command: SendMessageCommand,
     notification: NotificationEntity,
-    logCodeEnum: LogCodeEnum
+    logCodeEnum: LogCodeEnum,
+    error?: any
   ) {
-    await this.messageRepository.updateMessageStatus(message._id, status, null, errorId, errorMessage);
+    const errorString = (toStringify(error?.response?.body) ||
+      toStringify(error?.response) ||
+      toStringify(error) ||
+      JSON.stringify(errorMessageFallback)) as string;
+
+    if (error) {
+      Sentry.captureException(errorString);
+    }
```

Comment:
What do you think about the Sentry.captureException here? at the moment it will log only the once that passes the error param to this function (exception while sending a message).

Reply:
We will eventually will have to filterout the not relevant errors from sentry because it will create quite a lot of noise. But for now it's totally fine.